### DISCUSSION TITLE
双金了望周知

### DISCUSSION THREAD

**Discussion by Mollsity** [share121/inter-knot#5080] [原神]
本人将军单推人于9月17日6点整在《原神》中出了双金并且没歪感谢大家的祝福🥺🌹

**Comment by cuijiaqi0406** [comment]
我也，但是邦布池
![Uploading Screenshot_20240906_220327.jpg…]()

**Comment by KELERSAMA** [comment]
原本还打算抽卡的 现在不打算抽卡了 抽你就够了

**Comment by zdsy03** [comment]
吸收吸收
